build.gradle

```diff
@@ -12,7 +12,7 @@ repositories {
 
 
 dependencies {
-	compile 'net.java.dev.jna:jna:3.5.2'
+	compile 'net.java.dev.jna:jna:4.1.0'
```

Comment:
Unfortunately a higher version IS needed. According to the changelog 3.5.2 should support ARM but the binaries on mavencentral do not contain com/sun/jna/linux-arm/libjnidispatch.so. It is in 4.1.0.